src/electionguard/election_description_schema.json

```diff
@@ -0,0 +1,585 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "title": "Election Description",
+    "description": "Initial details to define an election",
+    "type": "object",
+    "properties": {
+        "name": {
+            "title": "Election Name",
+            "type": "object",
+            "properties": {
+                "text": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "value": {
+                                "title": "Value",
+                                "type": "string",
+                                "maxLength": 256
+                            },
+                            "language": {
+                                "title": "Language",
+                                "type": "string",
+                                "maxLength": 256
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "contact_information": {
+            "title": "Contact Information",
+            "type": [
+                "object",
+                "null"
+            ],
+            "properties": {
+                "address_line": {
+                    "title": "Full Address",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "string",
+                        "maxLength": 256
```

Comment:
Currently all the `string` have a limit of 256. We could just provide a default implementation with no limitations.